src/snippets/ternary.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Ternary Snippet
+---
+
+# Ternary
+
+Using Ternary in AssemblyScript
+
+```editor
+#!optimize=size&runtime=full
```

Comment:
Since the HTML does not utilize runtime helpers, we can have a much smaller wat file here by changing

```suggestion
#!runtime=half
```

Also, optimize defaults to `speed`, which yields the same result for small snippets and compiles a little faster due to skipping `--converge`.